src/neo/Network/P2P/Payloads/OracleResponse.cs

```diff
@@ -0,0 +1,48 @@
+using Neo.IO;
+using Neo.IO.Json;
+using System;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class OracleResponse : TransactionAttribute
+    {
+        private const int MaxResultSize = 1024;
+
+        public ulong Id;
+        public bool Success;
+        public byte[] Result;
+
+        public override TransactionAttributeType Type => TransactionAttributeType.OracleResponse;
+        public override bool AllowMultiple => false;
```

Comment:
If you want to send multiple request, we should't allow multiple Id's?

Reply:
This is `OracleResponse`. One transaction can send multiple requests. But one transaction has only one response.